content/en/docs/advanced-tasks/fail-forward-upgrades.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Fail forward upgrades"
+linkTitle: "Fail forward upgrades"
+weight: 4
+description: >
+    Fail forward upgrades
+---
+
+## What "Fail Forward" upgrades add
+A failed installation/upgrade is typically caused by one of two scenarios:
+1. **A Failed CSV:** The CSV is in the FAILED phase.
+2. **A Failed InstallPlan:** Usually occurs because a resource listed in the InstallPlan fails to be created or updated. An InstallPlan may fail independently of its CSV and may fail to create the CSV.
+
+By enabling "failed forward" upgrades, OLM will allow you to recover from failed installations and upgrades by:
```

Comment:
Isn't "enabling" and "opting into" the same action, or are there two steps?  This sounds very similar

Reply:
They're the same thing. Are you thinking I should just change this to "opting into"?

Reply:
They're the same thing, I think it makes sense to git rid of the first bullet to your point.